installer/install-playos/install-playos.py

```diff
@@ -154,25 +154,26 @@ def create_partitioning(device):
     return (disk)
 
 
-def _compute_geometries(device):
+def _compute_geometries(device, part_sizes):
     sectorSize = device.sectorSize
     esp = parted.Geometry(
         device=device,
         start=parted.sizeToSectors(8, "MB", sectorSize),
-        length=parted.sizeToSectors(550, "MB", sectorSize))
+        # TODO: this could be computed from grub + RESCUE_SYSTEM size
+        length=parted.sizeToSectors(800, "MB", sectorSize))
```

Comment:
My concern about changing this everywhere is that at some point in the future we somehow assume the larger size and break systems with the still existing 550 MB ESP.

Use a parameter for this size, too, and only increase it for the test image?

Reply:
Can be done, but note that that the current 550MB is _also_ just a guestimate. But I agree that for completeness perhaps it's most clear if everything is explicit.

Reply:
Yes, it may be a guestimate, but if we keep it unchanged in real installations we are more likely to notice any regressions resulting from size mismatches.

And I understand we don't currently need to change it for real installations?

Reply:
98456e586893e7f